docs/knowledgebase/getting-started/index.md

```diff
@@ -59,8 +60,7 @@ configure `rustup`:
 ```bash
 # Install
 curl https://sh.rustup.rs -sSf | sh
-# Add the rust compiler and other tools to your PATH.
-# Make sure to add this to your shell startup script, too.
+# Configure
```

Comment:
Why?

Reply:
I do not think it's necessary to dive into the details of what this does. Furthermore, I don't believe it's necessary to add this to the startup script.

Reply:
You source that file manually in every terminal you want to use rust?

Reply:
Sorry I just realized I never responded to this comment. I'm not sure if the installation scripts have been updated or something but it's no longer necessary to manually add the command to the shell startup script. I have not added such a command, nor do I need to source the file manually in ever terminal. I reviewed all the installation instructions for Rustup that I could find and I was not able to see this listed anywhere as a requirement https://rust-lang.github.io/rustup/installation/index.html